docs/deploy/index.mdx

```diff
@@ -1,52 +1,32 @@
 ---
-title: Deployment overview
+title: Deployment Overview
 description: Deployment overview page
 sidebar_position: 0
 ---
 
-# Tacetve a gemit quisquam praestat meruisse gradientis
+# Deploying to Weaviate
```

Comment:
Is "Deploying to Weaviate" the correct terminology? 
WDYT about "Deploying Weaviate" or "How to deploy Weaviate".